client/my-sites/stats/jetpack-upsell-section/use-purchased-products.tsx

```diff
@@ -7,25 +7,14 @@ import {
 	JETPACK_VIDEOPRESS_PRODUCTS,
 } from '@automattic/calypso-products';
 import { useEffect, useState } from 'react';
+import wpcom from 'calypso/lib/wp';
 
 //
 // WARNING: This hook will only work within Odyssey Stats!
 // It depends on api_root and nonce from window.configData!
```

Comment:
Is this line of comments necessary since we have removed the dependency for the `window.configData`?